### DISCUSSION TITLE
composefs is applying to the CNCF as a Sandbox project!

### DISCUSSION THREAD

**Discussion by cgwalters** [composefs/composefs#397] [Announcements]
We're excited to announce that composefs [is submitted](https://github.com/cncf/sandbox/issues/311) to the [CNCF](https://cncf.io) as a sandbox project!

While the composefs "core" is a generic verified filesystem mechanism, and will remain so, we are actively working to create bridges with OCI and other components in the ecosystem.

We're looking forward to building a flexible verified filesystem together for operating systems to containers!

More links:

- https://github.com/cncf/sandbox/issues/311
- https://www.redhat.com/en/blog/red-hat-contribute-comprehensive-container-tools-collection-cloud-native-computing-foundation
- https://blog.podman.io/2024/11/podman-buildah-and-skopeo-and-the-cloud-native-computing-

**Comment by hsiangkao** [comment]
Good to hear! 👍

**Reply by jluebbe** [reply]
Sounds good! 🚀

**Comment by cgwalters** [comment]
We're accepted! https://github.com/cncf/sandbox/issues/311#issuecomment-2595023225

**Comment by cgwalters** [comment]
https://github.com/cncf/sandbox/issues/340

**Comment by cgwalters** [comment]
Did someone reserve https://github.com/composefs ?

**Reply by allisonkarlitskaya** [reply]
Yup :)

**Reply by cgwalters** [reply]
OK, cool! Let's go ahead and use that as the org for the donation? Can you give me repo creation privs in that org?

**Reply by giuseppe** [reply]
please add me too

**Reply by cgwalters** [reply]
Done, though I think what *should* happen here is after the CNCF takes over ownership of the org, we'll start auto-syncing with the MAINTAINERS file AIUI
